plugins/scaffolder-field-extensions/scaffolder-frontend-module-http-request-field/README.md

```diff
@@ -79,3 +79,135 @@ The example template would result in the following dropdown:
 
 ![Alt text](images/dropdown_sample_closed.png?raw=true 'Example of the custom scaffolder field')
 ![Alt text](images/dropdown_sample_opened.png?raw=true 'Example of the custom scaffolder field')
+
+The simplest version of this parameter would look like this:
+
+````yaml
+---
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: http-param-example
+spec:
+  owner: roadie
+  type: service
+  parameters:
+    properties:
+      custom-basic:
+        type: string
+        ui:field: SelectFieldFromApi
+        ui:options:
+          path: 'catalog/entities'
+          # This assumes the api response returns an array of objects of ```{ metadata: { name: "", ... }, ... }```
+          valueSelector: 'metadata.name'
+````
+
+### Dynamic Select
+
+You can configure your http select to toggle between different requests in order to populate the dropdown depending on previously selected parameters.
+
+There are several ways to do this. The simplest looks like so:
+
+```yaml
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: custom-field-demo
+  title: Custom Field Demo
+  description: Custom Field Demo.
+spec:
+  owner: backstage/techdocs-core
+  type: service
+
+  parameters:
+    properties:
+      facetValue:
+        type: string
+        enum: ['kind', 'apiVersion']
+      custom:
+        type: string
+        # Use `SelectFieldFromApi` to configure the select field for the entry.
+        ui:field: SelectFieldFromApi
+
+        ui:options:
+          title: My Dropdown title
+          description: My custom description for the component
+          # The Path on the Backstage API and the parameters to fetch the data for the dropdown
+          path: 'catalog/entity-facets'
+          # This selects the array element from the API fetch response. It finds the array with the name kind
+          # under the facets object
+          arraySelector: 'facets.kind'
+          # (Optional) This selects the field in the array to use for the value of each select item. If its not specified
+          # it will use the value of the item directly.
+          valueSelector: 'count'
+          # (Optional) This selects the field in the array to use for the label of each select item.
+          labelSelector: 'value'
+          # (Optional)
+          dynamicParams:
+            # (Optional) This is the key of an additional dynamic query parameter that can be added to the request
+            paramKeyLocation: 'facet'
+            # (Optional) This is the key of a previous property who's selected value will be used as an additional query parameter value on the request
```

Comment:
This sentence is very complicated. I am not sure I actually understand what it means. Maybe the previous comment could do with simplying also. Maybe a suggestion: in the paragraph above this example, explain the scenario with concrete examples of data and then use them in this code example.